packages/adapters-library/src/defiProvider.ts

```diff
@@ -873,6 +883,7 @@ export class DefiProvider {
   async getSupport(input?: {
     filterChainIds?: Chain[] | undefined
     filterProtocolIds?: Protocol[] | undefined
+    includeProtocolTokens?: boolean
```

Comment:
With this new parameter added. Shouldn't it also use the new decorator?

Reply:
nope......this is a boolean to include protocol tokens it is not a address input